xml/libvirt_overview.xml

```diff
@@ -419,69 +252,231 @@
         </para>
       </listitem>
     </itemizedlist>
+
   </sect1>
-  <sect1 xml:id="libvirt-switch-daemons">
-    <title>Switching to modular daemons</title>
+  <sect1 xml:id="libvirt-monolithic-daemon">
+    <title>Starting and stopping the monolithic daemon</title>
 
     <para>
-      Several services need to be changed when switching from the monolithic to
-      modular daemons. It is recommended to stop or evict any running virtual
-      machines before switching between the daemon options.
+      The monolithic daemon is known as &libvirtd; and is configured via
+      <filename>/etc/libvirt/libvirtd.conf</filename>. &libvirtd; is managed
+      with several &systemd; unit files:
     </para>
 
-    <procedure>
-      <step>
+    <itemizedlist>
+      <listitem>
+        <para>
+          <emphasis>libvirtd.service</emphasis> - The main &systemd; unit file
```

Comment:
```suggestion
          <filename>libvirtd.service</filename> - The main &systemd; unit file
```

Reply:
IMO we should keep the <emphasis>. We're emphasizing an item name in an itemized list. It's also consistent with the next itemized list of service file settings.